railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -243,6 +249,17 @@ def config_target_version
     end
   end
 
+  class MasterGemfileGenerator < AppBuilder # :nodoc:
+    def install_shell_gemfile
+      template "GemfileMasterRailsInstaller", "Gemfile"
+      run 'bundle install'
+    end
+
+    def generate_edge_rails_app
+      run 'bundle exec rails new . --edge'
```

Comment:
Can you actually pass `.` and it'll generate into the current directory without naming the app "."? Think we also want to pass on extra options ala `--webpack=stimulus`.